.gitmodules

```diff
@@ -17,3 +17,6 @@
 [submodule "amundsencommon"]
 	path = amundsencommon
 	url = https://github.com/lyft/amundsencommon
+[submodule "amundseninfrastructure"]
```

Comment:
gitsubmodules is for submodule actual repos here. I don't think we need to add a new submodules givens it is mostly doc.

Reply:
> gitsubmodules is for submodule actual repos here. 
I'm new to using submodules, but, seemed to me that this would ultimately end up as a submodule. I just wasn't sure how to make it work from a PR. If you don't think it should be a submodule, I can remove it. lmk

>  I don't think we need to add a new submodules givens it is mostly doc.
I don't think it is mostly documentation. There should be several template files used to create infrastructure. Am I missing something?

Reply:
sorry for the wait, how about like superset(https://github.com/apache/incubator-superset/tree/master/contrib/docker)? We create an folder like `contrib/infrastructure/k8s` ?
cc @jinhyukchang @bolkedebruin in case they have other ideas.

Reply:
@feng-tao I'm not sure the necessity of the contrib folder, but, I don't mind it either. Just lmk what you prefer, I just went with what we had internally for another k8s based project.

Reply:
contrib is for `contribute` which I borrowed from superset or Airflow :)